Makefile

```diff
@@ -391,7 +382,7 @@ endif
 endif
 endif
 
-CFLAGS += -DMRBOOM -DHAVE_IBXM -D_FORTIFY_SOURCE=0 -DPLATFORM=\"$(platform)\" -DGIT_VERSION=\"$(GIT_VERSION)\"
+CFLAGS += -DMRBOOM -D_FORTIFY_SOURCE=0 -DPLATFORM=\"$(platform)\" -DGIT_VERSION=\"$(GIT_VERSION)\"
```

Comment:
isnt this gonna break the music?

Reply:
Added back in different path as it's not part of libretro-common